src/types/isEmail.js

```diff
@@ -0,0 +1,23 @@
+export default email => {
+
+    if (!email) return false;
+
+    let splits = email.split("@");
+
+    // if only one split return false
+    if (splits.length < 2) return false;
+
+    // if the first part is smaller than a character
+    if (splits[0].length === 0) return false;
+    if (splits[1].length === 0) return false;
+    if (splits[splits.length - 1].length === 0) return false;
+
+    splits = splits[1].split(".");
```

Comment:
You could argue that this doesn't happen often, but a (sub)domain is not a requirement for an email address to be valid. i.e. `tijn@nl` is technically valid and some TLD registrars/SMTP servers support this as well. Same goes for checking if the TLD length is at least 2. Just don't bother with anything after the `@`.